ansible-ipi-install/roles/shared-labs-prep/tasks/main.yml

```diff
@@ -285,6 +285,35 @@
       with_items: "{{ disable_nics }}"
   delegate_to: localhost
 
+- name: Create Manifest folder
+  block:
+    - name: Set manifest directory path
+      set_fact:
+        manifest_path: "{{ playbook_dir }}/roles/installer/files/manifests/"
+      
+    - name: Clean up any existing manifest content
+      file:
+        path: "{{ manifest_path }}"
+        state: absent
+
+    - name: Create manifest directory
```

Comment:
@mukrishn You seem to be deleting the entire manifests directory and recreating it, can you not just deletes the files inside of it so you don't need to recreate the manifests directory?